doc/topics/development/modules/developing.rst

```diff
@@ -48,8 +48,7 @@ a shell:
 
 .. code-block:: bash
 
-    # NOTE: use ipython2 instead of ipython for Arch Linux
-    ipython console --existing
+    jupyter console --existing
```

Comment:
Rereading this doc I still don't quite understand why `jupyter` install is required, could you elaborate a bit more?

Reply:
```
$ ipython3 console --existing
[TerminalIPythonApp] WARNING | Subcommand `ipython console` is deprecated and will be removed in future versions.
[TerminalIPythonApp] WARNING | You likely want to use `jupyter console` in the future
```